integration/integration_maven_build_test.go

```diff
@@ -0,0 +1,28 @@
+// +build integration
+// can be execute with go test -tags=integration ./integration/...
+
+package main
+
+import (
+	"testing"
+)
+
+func TestMavenBuildCloudSdkSpringProject(t *testing.T) {
+	t.Parallel()
```

Comment:
It is unclear to me whether this line can have any effect, unless you call Run() on the given T instance multiple times.